src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { LinkButton } from '../../components';
+import { Menu, Dropdown, Typography, Row, Col, Button } from 'antd';
+import { DownOutlined, UserOutlined } from '@ant-design/icons';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+const { Text } = Typography;
+
+const NavRow = styled(Row)`
+  background-color: white;
+  padding: 1em 0;
+`;
+const AntButtonLink = styled(LinkButton)`
+  color: black;
+  margin-left: 7em;
+`;
+const Logo = styled.img`
+  float: right;
+`;
+const LogoText = styled(Text)`
+  font-size: 24px;
+`;
+const UserDropdown = styled(Button)`
+  float: right;
+  margin-right: 3em;
```

Comment:
When I see this margin-right: 3em in two places next two each other, it makes me think "should this be a constant?"